lib/flagship.dart

```diff
@@ -38,23 +42,28 @@ class Flagship {
 
   Flagship._internal() {
     /// implement later
-    print("internal init");
   }
 
   /// Start Sdk
   ///
   /// envId : environement id (provided by flagship)
   /// apiKey: Api key (provided by flagship)
-  static start(String envId, String apiKey) {
-    _singleton.apiKey = apiKey;
-    _singleton.envId = envId;
-    print(
-        " ############# Start sdk  $envId  and  $apiKey   ######################");
+  static start(String envId, String apiKey, {FlagshipConfig? config}) {
+    if (FlagshipTools.chekcXidEnvironment(envId)) {
+      _singleton.apiKey = apiKey;
+      _singleton.envId = envId;
+      if (config != null) {
+        Flagship._configuration = config;
+      }
+      Flagship.logger(Level.INFO, STARTED);
+    } else {
+      Flagship.logger(Level.INFO, (INITIALIZATION_PARAM_ERROR));
```

Comment:
idem maybe error here now ?
actually maybe the SDK should fail here now ? not silently I mean.
Because if the apiKey & envId are wrong, then nothing will work?

Reply:
I thought about it, to do so, we need to add a Future to manage the error. (may be in the next iteration)